### PR TITLE
Removal of Attribute Editor Edit Buttons

### PR DESCRIPTION
**Pull request by capricornunicorn123**
### Prerequisites

- [x] Reviewed the [checklist](CHECKLIST.md)

- [ ] Reviewed feedback from the "Sonar Cloud" bot. Note that you have to wait
    for the "CI / Unit Tests") to complete first. Failed Unit tests can be 
    debugged by adding the label "verbose logging" to the GitHub PR.

### Description of the Change

Removed the Edit button from each AttributeTilePane as the same functionality was achievable my clicking in the value field. It was noted that indications of deactivated attributes was lost when the edit button was removed as the visual cue of deactivation was expressed through a color change to the edit button. This as rectified by deactivating the text field when an attribute was deactivated, giving an appropriate visual cue.  

Help Page Documentation was updated to reflect the new "edit-button-less" layout and the deactivated attribute visual cue.

### Alternate Designs
Considered expressing deactivated Attributes with "\<Disabled\>" or "\<Deactivated\>" text in the value field to express attribute deactivation. Opted against it due to complexity given varying attribute types such as icons and colors. Current solution is simpler and catches all scenarios. 

### Why Should This Be In Core?

It is an amendment to a core module. 

### Benefits

Cleaner UI promoting Intuitive Interaction.

### Possible Drawbacks

Indication of value editability is currently only expressed by the I-beam Pointer. This is a subtle indication and may be missed by some users initially. After the value is edited, the text field gains a neon blue boarder indicating the value is selected. This adds little value as once edited, users are likely to navigate out of the text field. If this boarder were expressed onMouseOver aswell as well as when selected, it would be a stronger prompt to show the value is editable.

### Verification Process

Tested Functionality creating and manipulating attributes.

### Applicable Issues

#1421

AttributeEditor

### CODE REVIEW

**Review by OrionsGuardian — APPROVED** [review]

**Review by Delphinus8821 — APPROVED** [review]

**Review by antares1470 — CHANGES_REQUESTED** [review]
Looks good. Are you possibly able to add an entry to `whatsnew.txt` to let users know of the change

**Review by antares1470 — APPROVED** [review]
This is good to go once the merge conflict is resolved

**Pull request merged**
